tools/check_format.py

```diff
@@ -705,6 +805,22 @@ def checkErrorMessages(error_messages):
   if args.add_excluded_prefixes:
     EXCLUDED_PREFIXES += tuple(args.add_excluded_prefixes)
 
+  # Returns the list of directories with owners listed in CODEOWNERS
+  def ownedDirectories():
+    owned = []
+    try:
+      with open("./CODEOWNERS") as f:
+        for line in f:
+          m = re.search(r'..*(extensions[^@]* )@.*@.*', line)
```

Comment:
Can you add a human description of what the regex does for those of us that are regex challenged?

Reply:
Done.
Note this does technically allow more than 2 owners (e.g. quic_listners) but does not allow 1 (reverse bridge filter) going forward.